public/scripts/gif.js

```diff
@@ -2,9 +2,10 @@ var imageLinks = [];
 var stampedImages = [];
 
 function getImageArray(){
-
-	
+	console.log("Here's a gif!");

```

Comment:
```suggestion
```

Do we need this debugging log here in the production code?

Reply:
Sorry, we don't need that in the production code. I merged my code with [https://github.com/a-mabe/tmvis](url) to get the features Abigail added and this was part of it. I will get rid of those logs.